docker-compose.yml

```diff
@@ -10,16 +17,42 @@ services:
             - "5432:5432"
   web:
     build: .
-    command: python3 manage.py runserver 0.0.0.0:8000
+    command: bash -c "if [ ! -d 'bluebutton-css' ] ; then git clone https://github.com/CMSgov/bluebutton-css.git ; else echo 'CSS already installed.' ; fi ; if [ '${BB20_ENABLE_REMOTE_DEBUG}' = true ] ; then python3 -m ptvsd --host 0.0.0.0 --port 5678 manage.py runserver 0.0.0.0:8000 --noreload ; else python3 manage.py runserver 0.0.0.0:8000; fi"
```

Comment:
The check for and clone of bluebutton-css is already handled in the Dockerfile, so should be removed here.

Reply:
got it

Reply:
FYI, I tried on linux and windows, the git clone CSS in dockerfile got executed but in final container of bb20, there is no bluebutton-css folder.